cves/CVE-2020-7471.yml

```diff
@@ -103,10 +103,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
```

Comment:
I would consider the answer to be true, because the offending code did have unit tests, but they were insufficient.

Reply:
That's fair